caddyhttp/gzip/responsefilter.go

```diff
@@ -25,6 +25,14 @@ func (l LengthFilter) ShouldCompress(w http.ResponseWriter) bool {
 	return l != 0 && int64(l) <= length
 }
 
+// SkipGzipped is ResponseFilter that will discard already gzipped files
```

Comment:
Yeah, I've added brotli support after initialy adding gzip support only. Name is misleading. I will change that :)